js/OmniNav.js

```diff
@@ -25,20 +107,207 @@ var OmniNav = (function() {
 
   var buildNav = function(target) {
     console.log("Building OmniNav for target: " + target);
```

Comment:
Should this console log remain here?

Reply:
Good question. We rarely ever have print statements in our code but I think this is actually an intentional one. It'll come in more handy when we go to implement it in blogs and inside where they have to be much more contextual. It's kind of an indication that a page is building the OmniNav dynamically (using this omninav-as-a-service) and that the requesting page is specifying the `target` correctly in the HTML.

For now we've just got the default target, nothing project-specific. We could look at removing the console.log in the future.